File: sentimental-mario-less/mario.py
# TODO
from cs50 import get_int
height = 0

while height < 1 or height > 8:
    height = get_int("height: ")

for i in range(height):
    for spaces in range(height-i-1):
        print(" ", end="")
    j = 0
    while j <= i:
        print("#", end="")
        j +=1
    print("")

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //i declare the variable height to get the height of the pyramid from the user
    int height;
    int dashes;
    int spaces;

    //keep asking the value of height while the number on the input is less than 1 or higher than 8
    do{
        height = get_int ("Height: ");

    }while (height < 1 || height > 8);

    for(int i = 0; i<= height; i++)
    {
        printf("#");
    }
    }